playground/backend/internal/executors/executor_builder.go

```diff
@@ -178,47 +188,47 @@ func (b *UnitTestExecutorBuilder) WithWorkingDir(dir string) *UnitTestExecutorBu
 	return b
 }
 
-//WithGraphOutput adds the need of graph output to executor
+// WithGraphOutput adds the need of graph output to executor
 func (b *UnitTestExecutorBuilder) WithGraphOutput() *UnitTestExecutorBuilder {
 	b.actions = append(b.actions, func(e *Executor) {
 		//todo
```

Comment:
What is the work remaining of this `//todo`?  Should it refer instead to a GitHub issue?